src/services/tunables/tunables_service.hpp

```diff
@@ -6,6 +6,8 @@
 
 namespace big
 {
+	constexpr int TUNABLE_BASE_ADDRESS = 262145; // This never changes
```

Comment:
I think the math might be wrong for this here. The Tunables globals starts at 0x40000, and the first tunable is at 0x40001. 

The way the tunables service generates the indexes is `*script_global(0x40000).at(i).as<int*>();`

So, shouldn't the base address be 0x40000?

Reply:
0x40000 (262144) will return offset + 1, which is wrong.
![1](https://github.com/user-attachments/assets/2446cdc0-5548-49b9-81a7-1d461af3571b)
This should be 0, not 1.
```c
Global_262145 = NETWORK::_NETWORK_GET_TUNABLES_REGISTRATION_FLOAT(joaat("CASH_MULTIPLIER"), 1f);
```